.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,9 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=11
```

Comment:
If that is for the Eclipse plugin, set version to 1.8 (cf., e.g., https://github.com/SeelabFhdo/lemma/blob/master/de.fhdo.lemma.data.avro/.settings/org.eclipse.jdt.core.prefs)